scripts/funcotator/testing/getGencodeGenesForVcfVariants.sh

```diff
@@ -20,7 +20,7 @@
 ###############################################################################
 
 #Setup variables for the script:
-UNALIASED_SCRIPT_NAME=$( readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}" )
+UNALIASED_SCRIPT_NAME=$( python -c "import os;print os.path.realpath(\"${BASH_SOURCE[0]}\")" )
```

Comment:
This prevents some issues due to calling the script from a random location if it is a symlink to another location.

I actually kind of hate calling python in there, but this works very well and all systems this will run on are guaranteed to have a base install of python so it's fine.